frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunDetails.tsx

```diff
@@ -85,7 +86,7 @@ const PipelineRunDetails: PipelineCoreDetailsPageComponent = ({ breadcrumbPath,
   }
 
   return (
-    <>
+    <PipelineRunDetailsVersionContextProvider resource={run}>
```

Comment:
I think we can limit this to wrap around `PipelineRunDrawerBottomContent` only since that's the only place the context seems to be used.

Reply:
Same goes for the `PipelineRunJobDetails` usage here.